README.md

```diff
@@ -185,6 +185,12 @@ removed.
   [resting state analysis from OMEGA datasets](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications)
   )
 
+## How to cite
+To credit BIDS-Matlab in your work, please cite the [Zenodo archive](https://zenodo.org/record/5910584), which points to a permanent record of all BIDS-Matlab versions (we encourage you to cite the specific version you used). Example citations (for bids-matlab 0.1.0):
+
+Gau, R., Flandin, G., Janke, A., Oostenveld, R., Madan, C., Niso, G., Szczepanik, M., Mutsaerts, H., Beliy, N., Norgaard, M., Pernet, C., & Phillips, C. bids-matlab (Version 0.1.0) Zenodo. doi: [10.5281/zenodo.5910585](https://doi.org/10.5281/zenodo.3363985)
```

Comment:
Realizing that there are some people who are in your list that do I have not added to the citation.cff

Gau, R., Flandin, G., Janke, A., Duval, T., Oostenveld, R., Madan, C., Niso, G., Szczepanik, M., Mutsaerts, H., Beliy, N., Norgaard, M., Pernet, C., Chrisophe, P., Shahzad, I.,  Botvinik-Nezer, R., Barilari, M., Battal, C., MacLean, M., & Caron-Guyon, J. bids-matlab (Version 0.1.0) [Computer software]. https://github.com/bids-standard/bids-matlab

Reply:
Thanks Remi! Please, edit as you see, I tried to take it from Zenodo, but not sure what the updated list is. 
For the text I just replicated a similar style of what there is on PyBIDS for "How to cite", as I think it's important to have it visible in the readme here as well. 
Also I see there are many references to PyBIDS in BIDS-Matlab page, but not many references to BIDS-Matlab in PyBIDS site, probably because it came first. But perhaps it could be nice to cross reference it there too.